soc/arm/atmel_sam/sam4l/soc.c

```diff
@@ -0,0 +1,299 @@
+/*
+ * Copyright (c) 2020 Gerson Fernando Budke <nandojve@gmail.com>
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief Atmel SAM4L MCU series initialization code
+ *
+ * This module provides routines to initialize and support board-level hardware
+ * for the Atmel SAM4L series processor.
+ */
+
+#include <device.h>
+#include <init.h>
+#include <soc.h>
+#include <arch/cpu.h>
+#include <arch/arm/aarch32/cortex_m/cmsis.h>
```

Comment:
not sure where you need this inclusion

Reply:
I'm trying to do similar things to another soc.c files. My feeling is, If I something need to be dropped here probably there is a bulk update for many soc files that need to be done too.

Reply:
Rem **#include <arch/arm/aarch32/cortex_m/cmsis.h>**